### ISSUE TITLE
what is full text dataset? 

### ISSUE THREAD

**Issue by janguck** [allenai/citeomatic#88]
Thank you for good paper, code

your team open dataset(title, abstract, year, author, venue, keyphrases, citation information), but what is the full text dataset? Have you ever created citation text for context aware?

Thanks.

**Comment by sergeyf** [comment]
Hello. We have not released any full text datasets in association with this paper & repo.

**Comment by janguck** [comment]
Yes! I know. I just wonder difference full text data and opened data

Thanks
